appscale/tools/remote_helper.py

```diff
@@ -108,6 +108,24 @@ def start_all_nodes(cls, options, count):
 
     return instance_ids, public_ips, private_ips
 
+  @classmethod
+  def enable_root_ssh(cls, options, public_ip):
+    """Enables root logins on the machine, enables SSH access,
```

Comment:
Could you change this sentence to -> Enables root logins and SSH access on the machine